src/Message.php

```diff
@@ -32,6 +32,9 @@ class Message
     private $audio = [];
 
     /** @var array */
+    private $files = [];
+
+    /** @var array */
     private $attachments = [];
```

Comment:
This is quite strange, if you see i've renamed the getter and setters for attachments to files, because "attachment" is the name of the botman base abstract class for attachments, and I don't know why the refactor added a new field instead of renaming the another one. We don't need anymore `$attachments` that now is unused.